pkgs/by-name/lu/lutgen-studio/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  makeWrapper,
+
+  fontconfig,
+  libGL,
+  libxkbcommon,
+  openssl,
+  wayland,
+  xorg,
+
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lutgen-studio";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "ozwaldorf";
+    repo = "lutgen-rs";
+    tag = "${pname}-v${version}";
+    hash = "sha256-Oog3qv+Pvo/Ib/fpoIcrExVkkcZj+tX8unH9LZod3YM=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-+8Sq6Jk914U0Vyr8cKLX++AF41j0R7y76ZG96SZJtxM=";
+
+  cargoBuildFlags = [
+    "--bin"
+    "${pname}"
+  ];
+  cargoTestFlags = [
+    "-p"
+    "${pname}"
+  ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  # Include dynamically loaded libraries
+  LD_LIBRARY_PATH = "${lib.makeLibraryPath ([
+    fontconfig
+    libGL
+    libxkbcommon
+    openssl
+    wayland
+    xorg.libXcursor
+    xorg.libXrandr
+    xorg.libXi
+    xorg.libX11
+  ])}";
+
+  postInstall = ''
+    wrapProgram "$out/bin/${pname}" \
+      --set LD_LIBRARY_PATH "${LD_LIBRARY_PATH}"
+  '';
+
+  passthru.updateScript = nix-update-script {
+    extraArgs = [ "--version-regex=^${pname}-v([0-9.]+)$" ];
+  };
+
+  meta = with lib; {
+    description = "Offical GUI for Lutgen, the best way to apply popular colorschemes to any image or wallpaper";
+    homepage = "https://github.com/ozwaldorf/lutgen-rs";
+    maintainers = with maintainers; [ ozwaldorf ];
+    mainProgram = "${pname}";
+    license = licenses.mit;
+  };
```

we should try to avoid `with lib;`
```suggestion
  meta = {
    description = "Offical GUI for Lutgen, the best way to apply popular colorschemes to any image or wallpaper";
    homepage = "https://github.com/ozwaldorf/lutgen-rs";
    maintainers = with lib.maintainers; [ ozwaldorf ];
    mainProgram = "${pname}";
    license = lib.licenses.mit;
  };
```